examples/with-tailwind/apps/web/next.config.ts

```diff
@@ -2,4 +2,10 @@
 module.exports = {
   reactStrictMode: true,
   transpilePackages: ["@repo/ui"],
+  typescript: {
```

Comment:
Same comment for docs!

Reply:
https://github.com/vercel/turborepo/pull/9764#discussion_r1927074231